examples/receiveLog.py

```diff
@@ -25,10 +25,12 @@
 
 if __name__ == "__main__":
     print "starting...\n"
-    x = events.EventReceiver("lsst8.ncsa.uiuc.edu", "LSSTLogging")
+    x = events.EventReceiver("lsst8.ncsa.illinois.edu", events.EventLog.LOGGING_TOPIC)
```

Comment:
I realise you're inheriting it from previous work, but `x` is not a very helpful variable name.